### PR TITLE
Add a Synonymous Form for service & image installs

### PR DESCRIPTION
**Pull request by thgreasi**
Change-type: minor
See: https://balena.zulipchat.com/#narrow/stream/346007-balena-io.2FbalenaCloud/topic/device.20installs.20image.2Fservice.20Synonymous.20Form

### CODE REVIEW

**Comment by thgreasi** [comment]
@balena-ci I self-certify!

File: src/balena-model.ts [hunk 1]
```diff
@@ -253,7 +254,7 @@ export interface Application {
 	uuid: string;
 	is_public: boolean;
 	application__has__env_var_name?: ApplicationEnvironmentVariable[];
-	application__has__name?: ApplicationConfigVariable[];
+	application__has__config_var_name?: ApplicationConfigVariable[];
```
Comment (Page-) [inline src/balena-model.ts:257]: Do we not have a test that verifies this file is up to date?

Comment (thgreasi) [reply src/balena-model.ts:257]: Looks like we don't. Let me add one while this gets merged.
Moreover I think that the generated property name is not correct either and that abstract-sql-to-typscript should have been using the Term Form when available, right?

Comment (thgreasi) [reply src/balena-model.ts:257]: Actually, we do have a test in bC, so let me copy it here.

**Pull request merged**